src/client/css/partials/settings.css

```
@@ -0,0 +1,158 @@
+.settings {
+  display: flex;
+  flex-direction: row;
+  padding: var(--padding-lg);
+}
+
+.settings-title,
+.settings-section-title {
+  color: var(--main-dark);
+}
+
+.settings-title {
+  flex: 1;
+}
+
+.settings-section-title {
+  margin: var(--padding-sm) 0 var(--padding-md) 0;
+}
+
+.settings-content {
+  color: var(--gray-40);
+  font-size: 0.875rem;
+  flex: 2;
+}
```

Suggestion: make this page responsive across all viewports rather than just for mobile media-query.  The following style will wrap the settings page sooner, rather than waiting for the breakpoint:

```suggestion
.settings {
  display: flex;
  flex-wrap: wrap;
  gap: var(--padding-md) var(--padding-xl);
  padding: var(--padding-lg);
}

.settings-title,
.settings-section-title {
  color: var(--main-dark);
}

.settings-section-title {
  margin: var(--padding-sm) 0 var(--padding-md) 0;
}

.settings-content {
  color: var(--gray-40);
  font-size: 0.875rem;
  flex: 1 1 66%;
}
```

---

Thanks for finding the layout issue and providing a great fix with this suggestion! :)